Mimicker/app/src/main/java/com/microsoft/mimickeralarm/utilities/VisionServiceRestClientNew.java

```diff
@@ -0,0 +1,117 @@
+package com.microsoft.mimickeralarm.utilities;
+
+import android.text.TextUtils;
+
+import com.google.gson.Gson;
+import com.microsoft.projectoxford.vision.VisionServiceClient;
+import com.microsoft.projectoxford.vision.contract.AnalyzeResult;
+import com.microsoft.projectoxford.vision.contract.OCR;
+import com.microsoft.projectoxford.vision.rest.VisionServiceException;
+import com.microsoft.projectoxford.vision.rest.WebServiceRequest;
+
+import org.apache.commons.io.IOUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+
+
+public class VisionServiceRestClientNew implements VisionServiceClient {
```

Comment:
I wonder if there wouldn't be better and much clearer if prefix `new` would be at the beginning = I mean `NewVisionServiceRestClient`.